dolphinscheduler-ui-next/src/views/projects/workflow/components/dag/use-cell-active.ts

```diff
@@ -89,29 +86,17 @@ export function useCellActive(options: Options) {
       PORT_SELECTED.groups[X6_PORT_OUT_NAME].attrs
     )
     const portDefault = _.cloneDeep(PORT.groups[X6_PORT_OUT_NAME].attrs)
-    const nodeHover = _.merge(_.cloneDeep(NODE.attrs), NODE_HOVER.attrs)
-    const nodeSelected = _.merge(_.cloneDeep(NODE.attrs), NODE_SELECTED.attrs)
 
-    let img = null
-    let nodeAttrs = null
     let portAttrs = null
-
     if (isHover || isSelected) {
```

Comment:
This code can be changed to 
```
if (isHover)
{
}
else if (isSelected)
{
}
else
{
}
```

Reply:
It has be handled in the style.
<img width="548" alt="image" src="https://user-images.githubusercontent.com/97265214/159484007-d2bc65ea-2530-4b25-ab29-6e870943a3b8.png">